### DISCUSSION TITLE
Change app selection in app tray upon middle click in waybar's "switch menu" option?

### DISCUSSION THREAD

**Discussion by H1manshu-Soni** [prasanthrangan/hyprdots#962] [Q&A] [answered]
If possible how can I change the app selection in the app tray which appears upon middle clicking the "switch menu" option in waybar
to simplify I will attach a SS.

![240226_00h49m25s_screenshot](https://github.com/prasanthrangan/hyprdots/assets/84933847/0ea00f79-8f68-405b-b228-1bcca429dc7d)

**Comment by RookiexCookie** [comment, marked as answer]
![image](https://github.com/prasanthrangan/hyprdots/assets/136296017/824f1186-4950-4ee2-ad08-5e8263a062ae)

you can define or remove the name of the apps in `~/.config/waybar/modules/wbar.jsonc`
